openapi.yaml

```diff
@@ -137,6 +205,42 @@ parameters:
       required:
         - data
       additionalProperties: false
+  financialAidPatchBody:
+    name: financialAidPatchBody
+    in: body
+    schema:
+      type: object
+      properties:
+        data:
+          properties:
+            id:
+              type: string
+              description: OSU ID of a person
+              pattern: '^\d{9}$'
+            type:
+              $ref: '#/definitions/FinancialAidType'
+            attributes:
+              properties:
+                onBaseDocumentType:
+                  type: string
+                  description: OnBase document type
+                  example: FA Last Day Attended
+                financialAidYear:
+                  type: string
+                  description: Search terms by financial aid year
```

Comment:
In the description we should point out the financialAidYear is actually two years. When I read the example `1819` I read that as the year 1819 and not 2018-2019

Reply:
Sounds good, fixed.